### PR TITLE
Injects http module dependencies with function call

### PR DESCRIPTION
**Pull request by dposada**
## Changes proposed in this PR

- instead of expecting the user to configure the http modules, allowing them to be passed programmatically

## Why are we making these changes?

To simplify the workflow for specifying custom http modules.

### CODE REVIEW

File: cli/cook/__main__.py [hunk 1]
```diff
@@ -36,4 +36,6 @@ def sigint_handler(_, __):
 signal.signal(signal.SIGINT, sigint_handler)
 
 if __name__ == '__main__':
+    import requests
+    http.inject_http_modules(requests, requests.adapters)
```
Comment (leifwalsh) [inline cli/cook/__main__.py]: Since these are defaulted in `configure()` anyway, I don't think you need to call this here.

Comment (dposada) [reply cli/cook/__main__.py]: Done

**Comment by dposada** [comment]
@leifwalsh I don't see `importlib` in `install_requires`

**Comment by leifwalsh** [comment]
Oh weird, I guess you were getting those transitively then.

**Comment by leifwalsh** [comment]
Thank you! Let me know when you’ve pushed a new sdist to PyPI.

**Comment by dposada** [comment]
@leifwalsh https://pypi.org/project/cook-client/2.7.0/

**Pull request merged**